File: Java/Item.java
package Java;

import java.time.LocalDateTime;

public class Item {
    private Integer id;
    private String name;
    private LocalDateTime manufactureTime;
    public Item(Integer id, String name, LocalDateTime manufactureTime) {
        this.id = id;
        this.name = name;
        this.manufactureTime = manufactureTime;
    }
    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public LocalDateTime getManufactureTime() {
        return manufactureTime;
    }
    public void setManufacturTime(LocalDateTime manufactureTime) {
        this.manufactureTime = manufactureTime;
    }
    @Override
    public String toString() {
        return "Item [id=" + id + ", manufactureTime=" + manufactureTime + ", name=" + name + "]";
    }

    
    
}

File: Kotlin/src/Item.kt
package Kotlin
import java.time.LocalDateTime
class Item(id:Int,name:String,manufactureTime:LocalDateTime) {
	var id:Int
		get() = this.id
		set(id:Int) {this.id = id}
	var name:String
		get() = this.name
		set(name:String) {this.name = name}
	var manufactureTime:LocalDateTime
		get() = this.manufactureTime
		set(manufactureTime:LocalDateTime) {this.manufactureTime = manufactureTime}
	init{
		this.id = id
		this.name = name
		this.manufactureTime = manufactureTime
	}
	
	override public fun toString():String{
		return "Item(id=$id,name=$name,manufactureTime=${manufactureTime.toString()})"
	}
}